test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -475,7 +483,27 @@ private void wipeCluster() throws Exception {
             waitForPendingRollupTasks();
         }
 
-        final Map<String, List<Map<?,?>>> inProgressSnapshots = wipeSnapshots();
+        // Clean up SLM policies before trying to wipe snapshots so that no new ones get started by SLM after wiping
+        deleteAllSLMPolicies();
+
+        SetOnce<Map<String, List<Map<?,?>>>> inProgressSnapshots = new SetOnce<>();
+        if (waitForAllSnapshotsWiped()) {
+            AtomicReference<Map<String, List<Map<?,?>>>> snapshots = new AtomicReference<>();
+            try {
+                // Repeatedly delete the snapshots until there aren't any
+                assertBusy(() -> {
```

Comment:
The 10s default worked fine in `SnapshotLifecycleIT`, the reason we were hitting problems was because the HLRC documentation test didn't include the cleanup code.  I'm fully open to increasing the timeout here, but I didn't see a good reason to do so yet. What do you think a good timeout would be?

Reply:
I'd just go with a minute at least (maybe just use 2 min here to be safe :)). Even if it got rare lately, a few GC pauses can easily add up to a few seconds and make such a complex operation here time out (we adjusted much simpler busy asserts from 10s to 1min lately).